Dockerfile

```diff
@@ -35,4 +35,4 @@ RUN cd mbofui && bower install && npm install && grunt --gruntfile Gruntfile-doc
 EXPOSE 8000
 
 # copy settings file and launch django
-CMD REMOTE_USER=bjensen gunicorn --workers=1 --bind=0.0.0.0:8000 hacks_mbof.wsgi:application
\ No newline at end of file
+CMD cp /usr/share/ocellus/settings.py ./hacks_mbof/; REMOTE_USER=bjensen gunicorn --workers=1 --bind=0.0.0.0:8000 hacks_mbof.wsgi:application
```

Comment:
Use an absolute path instead of `./hacks_mbof/`. Do these commands need to be on the same line?